frontend/src/lib/utils.tsx

```diff
@@ -825,7 +825,9 @@ export function autoCaptureEventToDescription(
     }
 
     const getValue = (): string | null => {
-        if (event.elements?.[0]?.text) {
+        if (event.properties.$el_text) {
+            return `${shortForm ? '' : 'with text '}"${event.properties.$el_text}"`
+        } else if (event.elements?.[0]?.text) {
             return `${shortForm ? '' : 'with text '}"${event.elements[0].text}"`
         } else if (event.elements?.[0]?.attributes?.['attr__aria-label']) {
             return `${shortForm ? '' : 'with aria label '}"${event.elements[0].attributes['attr__aria-label']}"`
```

Comment:
considered suggesting this since I find that original hard to parse, but I don't think this is necessarily better	

```suggestion
        const isAriaLabel = event.elements?.[0]?.attributes?.['attr__aria-label']
        const prefix = shortForm ? '' : isAriaLabel ? 'with aria label ' : 'with text '
        
        let value = null
        if (isAriaLabel) {
            value = event.elements?.[0]?.attributes?.['attr__aria-label']
        } else if (event.properties.$el_text) {
            value = event.properties.$el_text
        } else {
            value = event.elements?.[0]?.text
        }
        
        return value ? `${prefix}"${value}"` : null
```

Reply:
or this

```suggestion
        const text = event.properties.$el_text || event.elements?.[0]?.text
        const isAriaLabel = !text && event.elements?.[0]?.attributes?.['attr__aria-label']
        const prefix = shortForm ? '' : isAriaLabel ? 'with aria label ' : 'with text '
        const value = isAriaLabel ? event.elements?.[0]?.attributes?.['attr__aria-label'] : text

        return value ? `${prefix}"${value}"` : null
```